guacamole/src/main/frontend/src/app/index/indexModule.js

```diff
@@ -17,22 +17,37 @@
  * under the License.
  */
 
+require('angular-module-shim.js');
+require('relocateParameters.js');
+
+require('angular-translate-interpolation-messageformat');
+require('angular-translate-loader-static-files');
+
 /**
  * The module for the root of the application.
  */
 angular.module('index', [
+
+    require('angular-route'),
+    require('angular-translate'),
+
     'auth',
     'client',
     'clipboard',
     'home',
     'login',
     'manage',
     'navigation',
-    'ngRoute',
-    'ngTouch',
     'notification',
-    'pascalprecht.translate',
     'rest',
     'settings',
+
```

Comment:
Why is there a space before this last one?

Reply:
I thought it made sense to visually separate things into three groups:

* AngularJS modules from external libraries
* AngularJS modules we define ourselves with actual code
* AngularJS modules that are generated at build time (just `templates-main`)

Reply:
Makes sense to me.